### DISCUSSION TITLE
2005/12/06/%E6%88%91%E7%9A%84%E8%A1%A5%E7%89%99%E7%BB%8F%E5%8E%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#64] [Announcements]
# 2005/12/06/%E6%88%91%E7%9A%84%E8%A1%A5%E7%89%99%E7%BB%8F%E5%8E%86

我是大概在3个月前补了一次牙。

https://blog.qizhen.xyz/2005/12/06/%E6%88%91%E7%9A%84%E8%A1%A5%E7%89%99%E7%BB%8F%E5%8E%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

men
2005年12月15日 01:25 编辑
太晚了捆死了~~我该睡觉了~~明天再继续光顾你吧！看了一圈全是跋牙~~~请问”老头“你今年贵庚啊~~~~嘿嘿！！

回复
Jay
2005年12月7日 23:03 编辑
呵呵，等你老了，你就惨了
